content/docs/config-files.md

```diff
@@ -227,7 +236,7 @@ If you want to add **Function List** capability for your User Defined Language (
             </function>
         </parser>
 
-    where the `fn_udl_example` must match the `<association id>`.  The `displayName` sets what shows in the **Function List** window header.  The `...Expr` values are all defined in [regular expression syntax](../searching/#regular-expressions).
+    where the `fn_udl_example` must match the `<association id>` from the association map.  The `displayName` sets what shows in the **Function List** window header.  The `...Expr` values are all defined in [regular expression syntax](../searching/#regular-expressions).  See [the Function Parser documentation](../function-list/#function-parser) for more.

```

Comment:
For this sentence:
`where the `fn_udl_example` must match the `<association id>` from the association map.`
I would change it to:
`where the `fn_udl_example` for v7.9 and previous versions, it must match the `<association id>` from the association map; for v7.9.1 and later versions, feel free to name it as you want.`